library/src/main/res/layout/standard_list_item_view.xml

```diff
@@ -57,4 +57,22 @@
 			android:visibility="gone"
 		/>
 	</LinearLayout>
+
+	<!-- View for spacing chevron ImageView to the end -->
+	<View
+		android:layout_width="0dp"
+		android:layout_height="0dp"
+		android:layout_weight="1"
+		android:importantForAccessibility="no"
+		android:visibility="invisible"/>
```

Comment:
I hate that the rest of the file is wrongly formatted, but we need to be consistent